versioning/lockfiles/evolving.rst

```diff
@@ -51,6 +51,39 @@ compute from it the configuration specific profiles.
 These partial lockfiles will be smaller than the original app lockfiles, not containing information at all about
 ``app`` and ``libc``.
 
+Unlocking packages with --build
++++++++++++++++++++++++++++++++
+
+It is also possible to derive a partial lockfile for `libb/1.0`` without cloning the ``libb`` repository, directly with:
+
+.. code:: bash
+
+    $ conan lock create --reference=libb/1.0 --lockfile=app_release.lock --lockfile-out=libb_release.lock
+    $ conan lock create --reference=libb/1.0 --lockfile=app_debug.lock --lockfile-out=libb_debug.lock
+
+These new lockfiles could be used to install the ``libb/1.0`` package, without building it, but if we tried to
+build it from sources, it will fail:
+
+.. code:: bash
+
+    $ conan install libb/1.0@ --lockfile=libb_release.lock # Works
+    $ conan install libb/1.0@ --build=libb --lockfile=libb_release.lock # Fails, libb is locked
+
+The second scenario fails, because when we captured *app_release.lock* lockfile, it was completely locking all the
+information, including the package revision of ``libb/1.0``. If we try to build a new binary, the lock protection will
```

Comment:
Shorter sentences
```suggestion
The second scenario fails. This is because when the *app_release.lock* lockfile was captured, it completely locked all the
information (including ``libb/1.0``'s package revision). If we try to build a new binary, the lock protection will
```